spring/src/main/java/com/example/spring/controller/Deck.java

```diff
@@ -1,11 +1,10 @@
-package com.example.spring.Controller;
+package com.example.spring.controller;
 
 public class Deck {
     private int tier;
     private String deck;
 
     public Deck(int Tier, String deck) {
```

Comment:
この変数`Tier`は使っていないので、消してあげて良さそうですね。
```suggestion
    public Deck(String deck) {
```